node/lib/task.ts

```diff
@@ -591,6 +591,47 @@ export function exist(path: string): boolean {
 }
 
 /**
+ * Reads a directory's contents.
+ * see [fs.readdirsync](https://nodejs.org/api/fs.html#fs_fs_readdirsync_path_options)
+ * 
+ * @param       path    path of directory to readdirSync
+ * @returns     array of directory contents
+ */
+export function readdirSync(path: string): string[] {
+    return fs.readdirSync(path);
+}
+
+/**
+ * Deletes the specified file.
+ * see [fs.unlinkSync](https://nodejs.org/api/fs.html#fs_fs_unlinksync_path)
+ * 
+ * @param        path   path of the file to delete
+ */
+export function unlinkSync(path: string): void {
+    fs.unlinkSync(path);
```

Comment:
didn't see that method before; switched to using rmRF and pulled readdirSync out.